syntax = "proto2";
option optimize_for = LITE_RUNTIME;
option java_package = "ProtoMessages";
option java_outer_classname = "MessageResponsePlayerInfoBase";

message MessageResponsePlayerInfo {
    required int64 playerId = 1 [default = 0];
    required string playerName = 2 [default = ""];
    required int32 playerWins = 3 [default = 0];
    required int32 playerLoses = 4 [default = 0];
    required int32 ratingPosition = 5 [default = 0];
    required bytes playerAvatar = 6 [default = "\0"];
}